src/html-purify.js

```diff
@@ -92,7 +95,7 @@ See the accompanying LICENSE file for terms.
             break;
 
         case derivedState.TransitionName.ATTR_TO_AFTER_ATTR:
-            this.attrVals[attributeName] = '';
+            this.attrVals[attributeName] = null;
```

Comment:
@adon-at-work @neraliu should we also initialize attributeValue to null as opposed to '' at https://github.com/yahoo/context-parser/blob/master/src/context-parser.js#L43 ?